packages/cyberstorm/src/components/PaginationButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { MouseEventHandler, ReactNode } from "react";
+import styles from "./componentStyles/PaginationButton.module.css";
+
+export interface PaginationButtonProps {
+  isSelected?: boolean;
+  label?: string;
+  leftIcon?: ReactNode;
+  rightIcon?: ReactNode;
+  colorScheme?: "default";
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+}
+
+export const PaginationButton: React.FC<PaginationButtonProps> = (props) => {
+  const { label, leftIcon, rightIcon, colorScheme, onClick, isSelected } =
+    props;
+  return (
+    <button
+      type="button"
+      className={`${styles.root} ${getStyle(colorScheme, isSelected)}`}
+      onClick={onClick}
+    >
+      {leftIcon}
+      {label ? <div className={styles.label}>{label}</div> : null}
+      {rightIcon}
+    </button>
```

Comment:
To correct myself, while `aria-current="true"` is perfectly valid, more accurate value would be `aria-current="page"`, since that's what the item indicates - the current page in the pagination list we're on.

Reply:
Yup, I think I got it now 👌

Reply:
The notion about anchor vs button is up for debate, but the notion regarding the ellipsis items still stands.

Reply:
It's now in <span> element.